data/data-pipeline/data_pipeline/etl/sources/epa_rsei_aggregate/etl.py

```diff
@@ -0,0 +1,172 @@
+            self.TRACT_INPUT_COLUMN_NAME,
+            self.NUMBER_FACILITIES_INPUT_FIELD,
+            self.NUMBER_RELEASES_INPUT_FIELD,
+            self.NUMBER_CHEMICALS_INPUT_FIELD,
+            self.AVERAGE_TOXICITY_INPUT_FIELD,
+            self.SCORE_INPUT_FIELD,
+            self.POPULATION_INPUT_FIELD,
+            self.CSCORE_INPUT_FIELD,
+            self.NCSCORE_INPUT_FIELD,
+        ]
+        # The subsequent step make the current columns the first row
+        # of the dataframe and adds it as an individual sample
+        first_row = dict(zip(input_columns, self.df.columns))
+        self.df.columns = input_columns
+        # add first row to dataframe
+        row_df = pd.DataFrame(first_row, index=[0]).reset_index(drop=True)
+        # simply concatenate both dataframes
+        self.df = pd.concat([row_df, self.df])
+
+        score_columns = [x for x in self.df.columns if "SCORE" in x]
+
+        # coerce dataframe type to perform correct next steps
+        self.df[score_columns] = self.df[score_columns].astype(float)
+
+        self.df.rename(
+            columns={
+                self.TRACT_INPUT_COLUMN_NAME: self.GEOID_TRACT_FIELD_NAME,
+                self.NUMBER_FACILITIES_INPUT_FIELD: field_names.EPA_RSEI_NUMBER_FACILITIES_OUTPUT_FIELD,
+                self.NUMBER_RELEASES_INPUT_FIELD: field_names.EPA_RSEI_NUMBER_RELEASES_OUTPUT_FIELD,
+                self.NUMBER_CHEMICALS_INPUT_FIELD: field_names.EPA_RSEI_NUMBER_CHEMICALS_OUTPUT_FIELD,
+                self.AVERAGE_TOXICITY_INPUT_FIELD: field_names.EPA_RSEI_AVERAGE_TOXICITY_OUTPUT_FIELD,
+                self.SCORE_INPUT_FIELD: field_names.EPA_RSEI_SCORE_OUTPUT_FIELD,
+                self.CSCORE_INPUT_FIELD: field_names.EPA_RSEI_CSCORE_OUTPUT_FIELD,
+                self.NCSCORE_INPUT_FIELD: field_names.EPA_RSEI_NCSCORE_OUTPUT_FIELD,
+                self.POPULATION_INPUT_FIELD: field_names.EPA_RSEI_POPULATION_OUTPUT_FIELD,
+            },
+            inplace=True,
+        )
+
+        # Please note this: https://www.epa.gov/rsei/understanding-rsei-results#what
+        # Section: "What does a high RSEI Score mean?"
+        # Produce percentile rank for overall risk score
+        # This was created for the sole purpose to be used in the current
+        # iteration of Score L
+        self.df[field_names.EPA_RSEI_SCORE_PERCENTILE_RANK_FIELD] = self.df[
+            field_names.EPA_RSEI_SCORE_OUTPUT_FIELD
+        ].rank(
+            ascending=True,
+            pct=True,
+        )
+
+        # This threshold was arbitrarily chosen.
+        # It would make sense to enrich this with facilities, industries, or chemical
+        # that would enable some additional form of sub-stratification when examining
+        # different percentile ranges that are derived above.
+        self.df[field_names.EPA_RSEI_SCORE_THRESHOLD_FIELD] = (
+            self.df[field_names.EPA_RSEI_SCORE_PERCENTILE_RANK_FIELD]
+            > self.EPA_RSEI_SCORE_THRESHOLD_CUTOFF
```

Comment:
Nit: I think we use >= on most of these comparisons, not that it matters much in practice.